model/xgboost/dffml_model_xgboost/xdgregressor.py

```diff
@@ -0,0 +1,194 @@
+# Co-authored-by: John Andersen <johnandersenpdx@gmail.com>
+# Co-authored-by: Soren Andersen <sorenpdx@gmail.com>
+import pathlib
+from typing import AsyncIterator
+import importlib
+
+import joblib
+import pandas as pd
+from sklearn.metrics import r2_score
+import xgboost as xgb
+
+from dffml.base import config, field
+from dffml.model.accuracy import Accuracy
+from dffml.model.model import SimpleModel, ModelNotTrained
+from dffml.feature.feature import Feature, Features
+from dffml.source.source import Sources
+from dffml.util.entrypoint import entrypoint
+from dffml.record import Record
+
```

Comment:
We should follow the import guidelines https://intel.github.io/dffml/master/contributing/style.html#imports

Reply:
Which import guidelines am I breaking in particular, I noticed that I wasn't doing them in an XMAS structure so I have changed that. Is there anything else?

Reply:
No, everything else is good to go!